### DISCUSSION TITLE
a bit laggy

### DISCUSSION THREAD

**Discussion by yinjg1997** [he3-app/he3-feedback#152] [Bug Report]
### Version
He3-app 1.3.2

### Environment
Windows v1.3.2

### Steps to reproduce
Setting the low network mode to use He3 can reproduce this issue, and it cannot even be installed in a network free environment

### What is actually happening?

**Comment by lyzhang1999** [comment]
Thanks for your feedback, I have report this issue to product team: https://github.com/he3-app/he3-feedback/issues/153
